torch/testing/_internal/common_methods_invocations.py

```diff
@@ -8029,6 +8050,31 @@ def reference_flatten(input, start_dim=0, end_dim=-1):
                # UserWarning not triggered : Resized a non-empty tensor but did not warn about it.
                DecorateInfo(unittest.expectedFailure, 'TestCommon', 'test_out_warning'),
            )),
+    OpInfo('uniform',
+           op=lambda inp, *args, **kwargs: wrapper_set_seed(torch.Tensor.uniform_, inp, *args, **kwargs),
+           method_variant=None,
+           inplace_variant=torch.Tensor.uniform_,
+           dtypes=floating_and_complex_types_and(torch.bfloat16, torch.float16),
+           supports_out=False,
+           supports_autograd=False,
+           is_factory_function=False,
+           sample_inputs_func=sample_inputs_uniform,
+           error_inputs_func=error_inputs_uniform,
+           skips=(
+               # FX failed to normalize op - add the op to the op_skip list.
+               DecorateInfo(unittest.expectedFailure, 'TestNormalizeOperators', 'test_normalize_operator_exhaustive'),
+               # Tests that assume input tensor has a meningful effect on output tensor
+               DecorateInfo(unittest.expectedFailure, 'TestCommon', 'test_variant_consistency_eager'),
+               DecorateInfo(unittest.expectedFailure, 'TestMathBits', 'test_neg_view'),
+               DecorateInfo(unittest.expectedFailure, 'TestMathBits', 'test_conj_view'),
+               DecorateInfo(unittest.expectedFailure, 'TestMathBits', 'test_neg_conj_view'),
+               # AssertionError: JIT Test does not execute any logic
+               DecorateInfo(unittest.expectedFailure, 'TestJit', 'test_variant_consistency_jit'),
+               # aten.uniform_.default - couldn't find symbolic meta function/decomposition
+               DecorateInfo(unittest.expectedFailure, 'TestProxyTensorOpInfo', 'test_make_fx_symbolic_exhaustive'),
+               # aten.uniform was not decomposed
+               DecorateInfo(unittest.expectedFailure, 'TestDecomp', 'test_quick'),
```

Comment:
@Lezcano, I'm getting this failure

```
__________________________________________________________________ TestDecompCPU.test_quick_uniform_cpu_float32 __________________________________________________________________
Traceback (most recent call last):
  File "/home/srossross/pytorch/test/test_decomp.py", line 351, in test_quick
    self.do_cross_ref(device, dtype, op, run_all=False)
  File "/home/srossross/pytorch/torch/testing/_internal/common_utils.py", line 942, in wrapper
    fn(*args, **kwargs)
  File "/home/srossross/pytorch/test/test_decomp.py", line 505, in do_cross_ref
    check_decomposed(aten_name)
  File "/home/srossross/pytorch/test/test_decomp.py", line 458, in check_decomposed
    self.assertTrue(
  File "/home/srossross/.conda/envs/pytorch2/lib/python3.8/unittest/case.py", line 765, in assertTrue
    raise self.failureException(msg)
AssertionError: False is not true : aten.uniform was not decomposed, saw calls for: aten.uniform_.default. If your op is  CompositeImplicitAutograd you should skip this test by updating CROSS_REF_EXCLUDE_SET.
```

is this expectedFailure the correct way to handle this or should I add it to `CROSS_REF_EXCLUDE_SET`?

Reply:
This should be an xfail, I believe.